Automattic/social-logos
# Add TikTok and Medium Icons

![](https://user-images.githubusercontent.com/2060580/105994773-5b737e80-60a8-11eb-80b2-7441ab58dc80.png)

### Testing Instructions

* `git checkout add/tiktok-medium`
* `pushd ./docs; python -m SimpleHTTPServer 8001; popd`
* Go to `http://localhost:8001` in your browser.
* Ensure the new icons appear on the preview.
